spring-boot-project/spring-boot-actuator-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -2086,6 +2086,20 @@
         "replacement": "management.trace.http.include",
         "level": "error"
       }
+    },
+    {
+      "name": "management.tracing.baggage.enabled",
+      "type": "java.lang.Boolean",
+      "description": "Whether to enable Micrometer Tracing baggage propagation.",
+      "defaultValue": true
+    },
+    {
+      "name": "management.tracing.propagation.type",
+      "defaultValue": "W3C"
+    },
+    {
+      "name": "management.tracing.sampling.probability",
+      "defaultValue": "0.10"
```

Comment:
Does the annotation processor not pick up this default automatically?

Reply:
Intellij told me that it doesn't :shrug:

Reply:
It appears to be wrong (https://docs.spring.io/spring-boot/docs/3.0.0-M5/reference/html/application-properties.html#application-properties.actuator.management.tracing.sampling.probability) so this can be reverted.

Reply:
Please check default values are produced by building on the command-line.